uscope/imagep/streams.py

```diff
@@ -364,6 +365,10 @@ def run(self):
         current_image = current_images[0]
 
         ipp = config.get_usc().ipp.pipeline_last()
+        current_plugins = [p["plugin"] for p in ipp]
```

Comment:
This is not philosophically how the current system works. Instead the intention is that config only contains a simple data structure (think JSON) and that configuration drives which plugins are required.

Reply:
Yes, I couldn't be sure about, here's logic, I should analyse the project and rethink for here

Reply:
Hello @JohnDMcMaster, I added the plugin functionality. I did not understand how exactly should I implement this part. I wrote this code according to https://docs.google.com/document/d/18Lcze718SK4ZbZlFLauQnipJrSRh27BItK7SEN_bmbw/edit this documentation, If you could explain your request to me, I can change the code according to you.